frontend/src/LineGraph.res

```diff
@@ -138,10 +138,12 @@ module Legend = {
         ("<div>" ++ labelHTML ++ "&ndash; </div>") ++
         ("<div>" ++ " <b>" ++ yHTML ++ "</b>" ++ "</div>" ++ "</div>"))
       }
+      // Index at which we added the mean (we could add other stats later)
+      let statsIndex = Array.length(data.series) / 2
```

Comment:
This looks so weird :D I was not familiar with the code, and it took me a long time to understand that `data.series` stores the colored lines and then all the grey means seen in the background.

Perhaps the `data.series` array could first be [partitionned](https://rescript-lang.org/docs/manual/latest/api/belt/array#partition) using `unit.label == "mean"` into the actual lines separated from the global stats (to clean up the insertion of the "Overall Stats" header on `idx == statsIndex`...) but then there are some issues with the access to the `rawData_`.

Anyway this doesn't matter if we can get rid of dygraph in a near future!

Reply:
I know this looks weird, and I clarified the code a little bit with a comment, but I don't feel like the partitioning will make the code look any cleaner. I'm working with the assumption that we'll be able to get rid of dygraph soon enough.

Reply:
Yes I agree!